Nos/Views/StoriesView.swift

```diff
@@ -7,84 +7,84 @@
     }
     
     var body: some View {
         VStack {
-            ScrollView(.horizontal) {
-                ScrollViewReader { proxy in
-                    HStack {
-                        ForEach(authors) { author in
-                            let index = authors.firstIndex(of: author)
-                            Button { 
-                                if let index {
-                                    currentAuthorIndex = index
-                                }
-                            } label: { 
-                                let avatar = AvatarView(imageUrl: author.profilePhotoURL, size: 54)
-                                    .padding(.vertical, 8)
-                                    .padding(.horizontal, 4)
-                                
-                                if let index, index == currentAuthorIndex {
-                                    avatar
-                                        .background(
-                                            Circle()
-                                                .stroke(LinearGradient.diagonalAccent, lineWidth: 3)
-                                                .frame(width: 58, height: 58)
-                                        )
-                                } else {
-                                    avatar
-                                }
-                            }
-                            .id(index)
-                            .onAppear {
-                                scrollViewProxy = proxy
-                            }
-                        }
-                    }
+            TabView(selection: $selectedAuthor) {
+                ForEach(authors) { author in
+                    AuthorStoryView(
+                        author: author,
+                        cutoffDate: $cutoffDate,
+                        showPreviousAuthor: { self.showPreviousAuthor(from: self.authors) },
+                        showNextAuthor: { self.showNextAuthor(from: self.authors) }
+                    )
+                    .tag(author)
                 }
             }
-            if let currentAuthor = authors[safe: currentAuthorIndex] {
-                AuthorStoryView(
-                    author: currentAuthor,
-                    cutoffDate: $cutoffDate,
-                    showPreviousAuthor: { self.showPreviousAuthor(from: self.authors) },
-                    showNextAuthor: { self.showNextAuthor(from: self.authors) }
-                )
-                .id(currentAuthorIndex) // TODO: Why doesn't it work without this!?
-            } else {
-                Text("Hello, world")
-            }
+            .tabViewStyle(.page(indexDisplayMode: .never))
         }
         .background(Color.appBg)
         .nosNavigationBar(title: .stories)
+        .task {
+            if selectedAuthor == Author(), let firstAuthor = authors.first {
+                selectedAuthor = firstAuthor
```

Comment:
Authomatically selects the first author in the query results so that we don't block the UI by selecting it synchronously in the init